### ISSUE TITLE
Repetitive click to open the email field options menu

### ISSUE THREAD

**Issue by BOHEUS** [twentyhq/twenty#7193]
**Scenario:**
1. Log in
2. Go to People
3. Hover on any email, click on pen icon
4. Hover on any email and click on three vertical dots once they show up

**Actual:** If this sub-menu hasn't been opened yet, it'll only appear once user clicks on it twice, after that it works normally

[test.webm](https://github.com/user-attachments/assets/d42b81d9-0edf-46dc-8899-c92c3468a62a)

**Expected:** Sub-menu must appear after first click

**Note:** No error in devtools or logs
**Note 2:** The same problem is with multiple values in Domain name in Companies

**Comment by sid0-0** [comment]
I've spent quite some time on this and all I've been able to come up with is that the cause is that (if you check the DOM element's onclick attribute) the button has noop registered onclick from this function:
```
function trapClickOnNonInteractiveElement(node) {
     node.onclick = noop;
}
```
Which does not get overriden by passing onClick (I tried using a normal html button which only logs on click).
No idea where this is from or how to fix this.
Any help is appreciated.
